client/post-editor/editor-sidebar/index.jsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import EditorDrawer from 'post-editor/editor-drawer';
+import EditorGroundControl from 'post-editor/editor-ground-control';
+import DraftList from 'my-sites/drafts/draft-list';
+import EditorSidebarHeader from './header';
+import SidebarFooter from 'layout/sidebar/footer';
+import EditorActionBar from 'post-editor/editor-action-bar';
+
+export default React.createClass( {
```

Comment:
Similar to the other new component, this should probably be a `Component` class.